trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/noise_geometric.py

```diff
@@ -18,102 +18,102 @@
+        # Calculation of the distance vector and emission angle
+        dist        = np.sqrt(altitude**2+(s-x0)**2)  
+        theta       = np.arctan(np.abs(altitude/(s-x0)))
+        flag        = (s-x0) < 0 
+        theta[flag] = np.pi - np.arctan(np.abs(altitude/(s-x0)))[flag]      
         
-    elif flyover==1:
+    elif flyover == True:
         
         #--------------------------------------------------------
         #------------------ FLYOVER CALCULATION -----------------
         #--------------------------------------------------------
         
         # Azimuthal angle is zero for flyover condition
-        phi=np.zeros(n_steps)    
+        phi   = np.zeros(n_steps)    
         theta = np.zeros(n_steps)  
         
-        #Lift-off position from the brake release    
+        # Lift-off position from the brake release    
         estimate_tofl = SUAVE.Methods.Performance.estimate_take_off_field_length
     
-        # defining required data for tofl evaluation S0
-        takeoff_airport = SUAVE.Attributes.Airports.Airport()        
+        # Defining required data for tofl evaluation S0
+        takeoff_airport      = SUAVE.Attributes.Airports.Airport()        
+        atmo                 = Data()
+        atmo.base            = Data()
+        atmo.base.atmosphere = analyses.atmosphere
+        
         S_0 = estimate_tofl(config,analyses,takeoff_airport)           
 
-        #Microphone position from the brake release point
-        x0= np.float(6500. - S_0)
+        # Microphone position from the brake release point
+        x0 = np.float(6500. - S_0)
         
-        #Calculation of the distance vector and emission angle
-        dist  = np.sqrt(altitude**2+(s-x0)**2)
-
-        for i in range(0, n_steps):
-            if (s[i]-x0)< 0.:
-                theta[i] = np.arctan(np.abs(altitude[i]/(s[i]-x0)))
-            else:
-                theta[i] = np.pi - np.arctan(np.abs(altitude[i]/(s[i]-x0)))        
+        # Calculation of the distance vector and emission angle
+        dist        = np.sqrt(altitude**2+(s-x0)**2)
+        theta       = np.arctan(np.abs(altitude/(s-x0)))
+        flag        = (s-x0) < 0 
+        theta[flag] = np.pi - np.arctan(np.abs(altitude/(s-x0)))[flag]       
         
-    else:
+    elif sideline == True:
         
         #--------------------------------------------------------
         #-------------------SIDELINE CALCULATION-----------------
         #--------------------------------------------------------        
         
-        theta = np.zeros(n_steps) 
-
-        z0 = 450.  #position on the z-direction of the sideline microphone (lateral coordinate)
-        y0 =   0.  #position on the y-direction of the sideline microphone (altitude coordinate)
-        
+        theta = np.zeros(n_steps)  
+        y0    = 450.  # position on the y-direction of the sideline microphone (lateral coordinate) 
```

Comment:
should this be a user-specified parameter along with z0 and x0?

Reply:
No, this is a certification point. They actually had more hard-coded things and I went it to make it more generic.